README.md

```diff
@@ -11,6 +11,10 @@ applications. For performance critical algorithms, optimized assembly versions a
 included for x86 and ARM.
 
 ## Quickstart for Amazon Linux 2
+
+AWS-LC’s libcrypto is a C library and needs a C compiler. AWS-LC's libssl is a
```

Comment:
Nit:  Should call out libdecript here until we get rid of it?

Reply:
It's gone